### ISSUE TITLE
Exception when missing versions from CO status

### ISSUE THREAD

**Issue by bostrt** [kxr/o-must-gather#6]
```
$ omg get co 
Traceback (most recent call last):
  File "/home/rdu/rbost/.local/bin/omg", line 11, in <module>
    load_entry_point('o-must-gather==0.6', 'console_scripts', 'omg')()
  File "/home/rdu/rbost/.local/lib/python3.6/site-packages/omg/cli.py", line 71, in main
    func(args)
  File "/home/rdu/rbost/.local/lib/python3.6/site-packages/omg/cmd/get_main.py", line 150, in get_main
    getout_func(rt, ns, res, a.output, show_type)
  File "/home/rdu/rbost/.local/lib/python3.6/site-packages/omg/cmd/get/co_out.py", line 22, in co_out
    vers = co['status']['versions']
KeyError: 'versions'
```

Here's the example ClusterOperator output from a real cluster (it was having Operator issues post-install):

```
---
apiVersion: config.openshift.io/v1
kind: ClusterOperator
metadata:
  creationTimestamp: "2020-07-08T15:06:05Z"
  generation: 1
  name: monitoring
  resourceVersion: "47942"
  selfLink: /apis/config.openshift.io/v1/clusteroperators/monitoring
  uid: 1445b684-adb8-42b9-80c8-217c2c0b84c1
spec: {}
status:
  conditions:
  - lastTransitionTime: "2020-07-08T15:11:16Z"
    status: "False"
    type: Available
  - lastTransitionTime: "2020-07-08T17:14:58Z"
    message: Rolling out the stack.
    reason: RollOutInProgress
    status: "True"
    type: Progressing
  - lastTransitionTime: "2020-07-08T16:44:03Z"
    message: 'Failed to rollout the stack. Error: running task Updating Alertmanager
      failed: waiting for Alertmanager Route to become ready failed: waiting for RouteReady
      of alertmanager-main: no status available for alertmanager-main'
    reason: UpdatingAlertmanagerFailed
    status: "True"
    type: Degraded
  - lastTransitionTime: "2020-07-08T17:14:58Z"
    message: Rollout of the monitoring stack is in progress. Please wait until it
      finishes.
    reason: RollOutInProgress
    status: "True"
    type: Upgradeable
  extension: null
  relatedObjects:
  - group: ""
    name: openshift-monitoring
    resource: namespaces
  - group: ""
    name: openshift-monitoring
    resource: all
  - group: monitoring.coreos.com
    name: ""
    resource: servicemonitors
  - group: monitoring.coreos.com
    name: ""
    resource: prometheusrules
  - group: monitoring.coreos.com
    name: ""
    resource: alertmanagers
  - group: monitoring.coreos.com
    name: ""
    resource: prometheuses

```

**Comment by kxr** [comment]
@bostrt Thank you very much for reporting this and providing the sample that caused the error. This is caused by the fact that I am blindly expecting `status.versions` and in this case, its not there and hence the `KeyError: 'versions'`. This will be fixed/handled in the next version.

**Comment by kxr** [comment]
@bostrt it's fixed in 0.8
